homeworks/11-animations/index.css

```diff
@@ -0,0 +1,98 @@
+* {
+  padding: 0;
+  margin: 0;
+  --delay: 180ms;
```

Comment:
Common approach is to use `:root` selector to define scope for CSS variables.

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#basic_usage